requirements.txt

```diff
@@ -8,6 +8,7 @@ ipywidgets>=7.5.1
 json5>=0.9.3
 jupyter>=1.0.0
 matplotlib==3.1.3
+networkx==2.8.8
 numba>=0.48.0
 numpy==1.18.3
```

Comment:
completely outdated (I modified the code in the branch `ramp_forecast` to use up to date versions of these libraries, I can make the PR and we merge this after if you prefer.

Reply:
removed